Gemfile

```diff
@@ -60,10 +60,13 @@ group :development do
   gem 'listen', '~> 3.9'
   gem 'lookbook', '~> 2.3.4'
   gem 'rack-mini-profiler'
-  gem 'rubocop', '~> 1.66', require: false
-  gem 'rubocop-performance', '~> 1.25', require: false
-  gem 'rubocop-rails', '~> 2.26', require: false
-  gem 'rubocop-rspec', '~> 2.25', require: false
+  gem 'rubocop', require: false
+  gem 'rubocop-capybara', require: false
+  gem 'rubocop-factory_bot', require: false
```

Comment:
I've removed the pessimistic locking since I don't think it really makes sense for a dev dependency - adds friction to updating without the same safety benefit for functional dependencies.

But I'm happy to add them back if desired